docs/shared/esg_grid.yaml

```diff
@@ -0,0 +1,22 @@
+esg_grid:
+  execution:
+    batchargs:
+      nodes: 1
+      stdout: /path/to/runscript.out
+      walltime: "08:00:00"
+    executable: /path/esg_grid
+    mpicmd: srun
```

Comment:
I think the notes doc for this component said that this is a serial (non-MPI) component, in which case we probably shouldn't provide a `mpicmd` in this example YAML.